packages/fxa-auth-db-mysql/lib/db/schema/patch-111-112.sql

```diff
@@ -0,0 +1,121 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+CALL assertPatchLevel('111');
+
+-- Explicitly set the collation on the email field, so that mysql will use
+-- the email table index and not do a full table scan.
+-- See https://github.com/mozilla/fxa/issues/6006.
+CREATE PROCEDURE `emailRecord_6` (
+    IN `inEmail` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_bin
+)
+BEGIN
+    SELECT
+        a.uid,
+        a.email,
+        a.normalizedEmail,
+        a.emailVerified,
+        a.emailCode,
+        a.kA,
+        a.wrapWrapKb,
+        a.verifierVersion,
+        a.authSalt,
+        a.verifierSetAt,
+        a.lockedAt,
+        a.createdAt
+    FROM
+        accounts a
+    WHERE
+        a.normalizedEmail = LOWER(inEmail)
+    ;
+END;
+
+-- Remove the eco_anon_id field from createAccount, which we really don't ever
+-- plan to use in practice. Also explicitly set collation on the normalized
+-- email field, so that mysql will use the index and not do a full table read.
+-- See https://github.com/mozilla/fxa/issues/6006.
+CREATE PROCEDURE `createAccount_9`(
+    IN `inUid` BINARY(16) ,
+    IN `inNormalizedEmail` VARCHAR(255) CHARACTER SET utf8 COLLATE utf8_bin,
+    IN `inEmail` VARCHAR(255),
```

Comment:
Noting Ryan's comment in https://github.com/mozilla/fxa/issues/6006#issuecomment-662201326
```
Warning: expected "utf8 COLLATE utf8_bin" for inEmail in createAccount_8!
IN `inEmail` VARCHAR(255),
```